packages/blockchain-wallet-v4-frontend/src/components/BuySell/Failure.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components'
+import { FormattedMessage } from 'react-intl'
+import { Text, Link } from 'blockchain-info-components'
+import { path } from 'ramda'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  width: 100%;
+  div:last-of-type {
+    margin-top: 10px;
+  }
+`
+
+const Loading = (e) => (
+  <Container>
+    <Text weight={300} size='16px'>
+      <FormattedMessage id='buysell.general.failure.message' defaultMessage='Sorry, an error has occurred while connecting to your exchange partner.' />
+    </Text>
+    <Text weight={300} size='16px'>
+      <FormattedMessage id='buysell.general.failure.message2' defaultMessage='If the problem continues, please reach out to our support team {supportLink}' values={{ supportLink: <Link href='https://support.blockchain.com/hc/en-us/requests/new' target='_blank' size='16px' weight={300}><FormattedMessage id='buysell.general.failure.here' defaultMessage='here.' /></Link> }} />
```

Comment:
I think it's better to make substitution a separate variable, too long line here, makes code unreadable